Simplified/NYPLCatalogGroupedFeedViewController.m

```diff
@@ -373,8 +373,15 @@ - (void)previewingContext:(__unused id<UIViewControllerPreviewing>)previewingCon
 
 - (void)entryPointViewDidSelectWithEntryPointFacet:(NYPLCatalogFacet *)entryPointFacet {
   NSURL *const newURL = entryPointFacet.href;
-  self.remoteViewController.URL = newURL;
-  [self.remoteViewController load];
+
+  if (newURL != nil) {
+    [self.remoteViewController loadWithURL:newURL];
+  } else {
+    [NYPLErrorLogger logErrorWithCode:NYPLErrorCodeNoURL
+                              context:NSStringFromClass([self class])
+                              message:@"Catalog facet missing href URL"
+                             metadata:nil];
+  }
```

Comment:
Is there feedback to the user on what happens if the URL is nil?

Reply:
Good catch, I added the Reload UI here.